### PR TITLE
Create consts for meta.pb filename.

### PR DESCRIPTION
**Pull request by feuerste**

### CODE REVIEW

File: xray/src/utils.rs [hunk 1]
```diff
@@ -1,12 +1,24 @@
...
 use std::path::{Path, PathBuf};
 
-const META_PREFIX: &str = "meta";
+lazy_static::lazy_static! {
+    static ref META_PREFIX: &'static str = Path::new(META_FILENAME)
+        .file_stem()
+        .unwrap()
+        .to_str()
+        .unwrap();
+    static ref META_SUFFIX: &'static str = Path::new(META_FILENAME)
```
Comment (krzesi-mir) [inline xray/src/utils.rs:13]: nit: `META_EXTENSION`?

File: xray/src/utils.rs [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
+        .extension()
+        .unwrap()
+        .to_str()
+        .unwrap();
+}
 
 pub fn get_meta_pb_path(directory: &Path, id: NodeId) -> PathBuf {
-    directory.join(&format!("{}.pb", id).replace(NODE_PREFIX, META_PREFIX))
+    directory.join(&format!("{}.{}", id, *META_SUFFIX).replace(NODE_PREFIX, *META_PREFIX))
```
Comment (krzesi-mir) [inline xray/src/utils.rs:21]: nit: how about
```suggestion
    directory.join(id.to_string().replace(NODE_PREFIX, *META_PREFIX)).with_extension(*META_SUFFIX))
```
?

File: xray/src/utils.rs (hunk 1 above)
Comment (feuerste) [reply xray/src/utils.rs]: Done

File: xray/src/utils.rs (hunk 2 above)
Comment (feuerste) [reply xray/src/utils.rs]: Done

**Comment by feuerste** [comment]
Thanks for your review!

**Pull request merged**